terraform/addons/private-registry/.header.md

```diff
@@ -0,0 +1,33 @@
+# Private Container Image Registry
+
+This addon will provision the correct IAM policy to attach to the Fleet config for the ECS task definition
+to utilize private registry credentials when pulling container images.
+
+## Using a private container image repository
+
+First create an AWS Secrets Manager Secret with your preferred method, for example:
+```shell
```

Comment:
Do you prefer this method over creating the secretsmanager secret as a terraform resource and then recommending either a secretsmanager secret version or setting the version on the created secret via the AWS CLI?

Reply:
I suppose I was conflicted about how to go about this. Since its not like a database secret that can be generated randomly, this needs to be supplied and I didn't want to have a strong opinion on how the user creates the secret (thus supplying the secret material).

Reply:
We can go forward with this as-is.  This way the secret can be created either via terraform or via the CLI.  The only complication would be if a custom KMS key were used on the user-created resource.  This is at least covers probably 95% of the use-case.